fenics/apt.txt

```diff
@@ -0,0 +1 @@
+build-essential
```

Comment:
It is important to use the same compiler which was used to initially build fenics, so we should use the conda compiler rather than the one from the Linux distribution.